From 7d08c5610115f97fe88e9517449af89ade5979bc Mon Sep 17 00:00:00 2001
From: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Date: Tue, 3 Dec 2019 01:03:53 -0800
Subject: [PATCH 0067/1851] firmware: xilinx: sync firmware pm_ioctl_id entries
 with PLM pm_ioctl_id entries

commit 665a19476e11cd1855deb4deeb5da889b92bb9ef from
https://github.com/Xilinx/linux-xlnx.git

This patch adds enum IOCTL_OSPI_MUX_SELECT into pm_ioctl_id present in
xlnx-zynqmp.h to be in sync with PLM pm_ioctl_id entries.

Signed-off-by: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Rajan Vaja <rajan.vaja@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/linux/firmware/xlnx-zynqmp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/firmware/xlnx-zynqmp.h b/include/linux/firmware/xlnx-zynqmp.h
index 880493692e62..d6cc1a9a5e87 100644
--- a/include/linux/firmware/xlnx-zynqmp.h
+++ b/include/linux/firmware/xlnx-zynqmp.h
@@ -170,6 +170,7 @@ enum pm_ioctl_id {
 	/* Probe counter read/write */
 	IOCTL_PROBE_COUNTER_READ,
 	IOCTL_PROBE_COUNTER_WRITE,
+	IOCTL_OSPI_MUX_SELECT,
 };
 
 enum pm_query_id {
-- 
2.31.1

